RELEASING.md

```diff
@@ -26,34 +26,34 @@ Change `**next**` in the "Stable Release" section in README that warns users tha
 ```
 ## Stable Release
 
-You're reading the documentation for the **stable** release of slack-ruby-bot-server, 0.2.2. See [UPGRADING](UPGRADING.md) when upgrading from an older version.
+You're reading the documentation for the **stable** release of slack-ruby-bot-server, 2.0.1. See [UPGRADING](UPGRADING.md) when upgrading from an older version.
 ```
 
 Commit your changes.
 
 ```
 git add README.md CHANGELOG.md
-git commit -m "Preparing for release, 0.2.2."
-git push origin master
+git commit -m "Preparing for release, 2.0.1."
+git push upstream master
```

Comment:
Typically I release by checking up upstream to avoid having any kind of fork pollution. Maybe let's say that above and change this back to `origin`? It's NBD anyway.

Reply:
Reverted.